2022 Review draft (#13)
ISISDataPolicy/policy: data_policy.md

# ISIS Data Policy  

> The ISIS data policy lays out the guarantees ISIS provides, and the requirements for users of the facilities, pertaining to the curation, access, ownership, use and storage of data collected at ISIS.

Reviewer: ```suggestion
The ISIS data policy lays out the guarantees ISIS provides, and the requirements for users of the facility, pertaining to the curation, access, ownership, use and storage of data collected at ISIS.
```

Reviewer: changed facilities to facility since most of our users may read facilities as multiple neutron and muon facilities

> The policy applies to anyone using ISIS facilities to produce experimental data, except as part of 'commercial-in-confidence' access to ISIS. Commercial users must instead come to an agreement about how their data is to be handled with ISIS before running their experiment.

Reviewer: ```suggestion
The policy applies to anyone using ISIS to produce experimental data, except as part of 'commercial-in-confidence' access to ISIS. Commercial users must instead come to an agreement about how their data is to be handled with ISIS before running their experiment.
```

> ISIS is committed to the principles of FAIR data in order to support research, and the advancement of knowledge and technology. We release data in common, easily accessible formats under a permissive Creative Commons license, along with detailed metadata in a variety of human and machine readable formats.

Reviewer: ```suggestion
ISIS is committed to support research and the advancement of knowledge and technology. We release data in easily accessible formats under a permissive Creative Commons license, along with metadata in a variety of human and machine readable formats.
```

Reviewer: I have removed the 'principles of FAIR' simply because if we mention FAIR, which not all our users know what is, we need to also define it and reference it. As needed Elizabeth, Abigail etc can advice on worded on this when we send these suggestions for changes round

Author: The rest of the line was only really there to support the 'principles for FAIR data' bit - I don't think it makes sense to leave it in without that. I'd suggest we either remove this line entirely, or make `FAIR` a link to an explanation (maybe https://www.go-fair.org/fair-principles/ or the wikipedia page?), as I agree that it's not good to just drop it in without explanation.

3.1.1 All raw data and associated metadata obtained as a result of free (non-commercial) access to ISIS, will be treated as described below, with ISIS acting as the custodian.

> 3.1.2 All raw data, facility generated reduced data and associated metadata obtained as a result of ‘commercial-in-confidence’ access to ISIS will be owned exclusively by the commercial user.  Commercial users must agree with their relevant instruments scientists how they wish their raw data, facility generated reduced data and metadata to be managed before the start of any experiment.

Reviewer: I agree it is preferable to delete 3.1.2 and replace it with a sentence in the introduction. But we should check with our industrial person Graham Appleby, including whether "Commercial users must agree with their relevant instruments scientists how they wish their raw data, facility generated reduced data and metadata to be managed before the start of any experiment" is obsolete or other. I will email Graham CC you in

Reviewer: Had meeting with Graham and Christ Frost

3.4.1 Raw data and metadata beyond the period that it is stored on instrument-related computers will long-term be accessible and searchable from the on-line catalogue.

3.4.2 Access to raw data obtained from an experiment is restricted to the experimental team for an embargo period of three years after the end of the experiment. Thereafter, it will become available to the wider community under the CC-BY license that obliges users to acknowledge ISIS and cite [Digital Object Identifiers (DOI)](https://www.isis.stfc.ac.uk/Pages/Digital-Object-Identifiers-%28DOIs%29-for-ISIS-Data.aspx) that are linked to ISIS data. Any PI that wishes their data to remain ‘restricted access’ for a longer period will be required to make a special case to the Director of ISIS.

> 3.4.3 Some elements of the metadata associated with an experiment are highlighted as being public during the proposal process, and will be publicly accessible as soon as the proposal is accepted. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.

Reviewer: ```suggestion
3.4.3 Some elements of the metadata associated with successful ISIS proposals are made publicly accessible as soon as the proposal is accepted such as the title of the proposal and the principle investigator. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.
```

Author: I'd like to keep it explicit that the 'source of truth' for which fields this applies to comes from the proposals system, so we don't have to update the policy if it changes. A midway point might be something like:

```suggestion
3.4.3 During the proposals process, some elements of the metadata associated with an experiment (such as the title) are indicated as being public; these will be publicly accessible immediately upon the proposal being accepted. All other elements of the associated metadata are handled in the same way as the raw experimental data in regards to the embargo period.
```
